### ISSUE TITLE
--version should not fail due to rclone.conf

### ISSUE THREAD

**Issue by kcwu** [ncw/rclone#2096]
> What is your rclone version (eg output from `rclone -V`)

rclone v1.39-DEV
- os/arch: freebsd/amd64
- go version: go1.9.3

> Which OS you are using and how many bits (eg Windows 7, 64 bit)

FreeBSD

> Which cloud storage system are you using? (eg Google Drive)

n/a

> The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

My rclone.conf is owned by root for some reason.
```
ls -l $HOME/.rclone.conf
-rw-------  1 root  kcwu  937 Feb 26 14:19 /home/kcwu/.rclone.conf
```

Then `rclone -V` will fail with normal user permission.
```
$ rclone -V
2018/02/26 14:55:38 Failed to load config file "/home/kcwu/.rclone.conf": open /home/kcwu/.rclone.conf: permission denied
```

> A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)

I expect "-V" should never fail for any reason.

**Comment by daniel-loader** [comment]
Sounds like a reasonable request, the binary should report a version without a config accessible I guess.
Makes extra sense when you consider my recent PR forces rclone to do a version declaration to print at the end of the script. In your example this would fail on a fresh install of the install.sh (assuming it can't write a default rclone.conf in the `~/.config/rclone/` path.

**Comment by ncw** [comment]
I agree, version should never fail.  I expect this to be annoyingly difficult to fix though as it will involve lots of shuffling init sequences about :-(
